src/main/java/org/jenkinsci/plugins/orgfolder/github/MainLogic.java

```diff
@@ -0,0 +1,204 @@
+                        String url = Jenkins.getActiveInstance().getRootUrl() + "github-webhook/";
+                        if (!existsHook(org, url)) {
+                            org.createWebHook(new URL(url), Arrays.asList(GHEvent.REPOSITORY,GHEvent.PUSH));
+                            LOGGER.log(Level.INFO, "A webhook was registered for the organization {0}", org.getHtmlUrl());
+                        }
+                        // keep trying until the hook gets successfully installed
+                        // if the user doesn't have the proper permission, this will cause
+                        // a repeated failure, but this code doesn't execute too often.
+                        orghook.on();
+                    } catch (IOException e) {
+                        LOGGER.log(Level.WARNING, "Failed to register GitHub Org hook to "+u.getHtmlUrl(), e);
+                    }
+                }
+
+
+                bc.commit();
+            } finally {
+                bc.abort();
+                UPDATING.get().remove(of);
+            }
+        }
+    }
+
+    /**
+     * Applies UI customizations to a level below {@link OrganizationFolder}, which maps to a repository.
+     */
+    public void applyRepo(WorkflowMultiBranchProject item, GitHubSCMNavigator scm) throws IOException {
+        if (UPDATING.get().add(item)) {
+            GitHub hub = connect(item, scm);
+            GHRepository repo = hub.getRepository(scm.getRepoOwner() + '/' + item.getName());
+
+            BulkChange bc = new BulkChange(item);
+            try {
+                item.setIcon(new GitHubRepoIcon());
+                item.getProperties().replace(new GitHubRepoProperty(repo));
+                item.replaceAction(new GitHubLink("repo",repo.getHtmlUrl()));
+                if (item.getView("Branches")==null && item.getView("All") instanceof AllView) {
+                    // create initial views
+                    ListView bv = new ListView("Branches");
+                    DescribableList<ListViewColumn, Descriptor<ListViewColumn>> cols = bv.getColumns();
+                    JobColumn name = cols.get(JobColumn.class);
+                    if (name!=null)
+                        cols.replace(name,new CustomNameJobColumn(Messages.class, Messages._ListViewColumn_Branch()));
+                    bv.getJobFilters().add(new BranchJobFilter());
+
+                    ListView pv = new ListView("Pull Requests");
+                    cols = pv.getColumns();
+                    name = cols.get(JobColumn.class);
+                    if (name!=null)
+                        cols.replace(name,new CustomNameJobColumn(Messages.class, Messages._ListViewColumn_PullRequest()));
+                    bv.getJobFilters().add(new PullRequestJobFilter());
+
+                    item.addView(bv);
+                    item.addView(pv);
+                    item.deleteView(item.getView("All"));
+                    item.setPrimaryView(bv);
```

Comment:
At least not until that feature comes to public repositories.
User can change this very easily too once it's created.